broker-core/src/test/java/io/zeebe/broker/system/BrokerHttpServerTest.java

```diff
@@ -62,4 +62,18 @@ public void shouldConfigureMetricsHttpServer() {
       // expect error
     }
   }
+
+  @Test
+  public void shouldGetReadyStatus() {
+    final MetricsCfg metricsCfg = brokerWithEnabledMetricsHttpServer.getBrokerCfg().getMetrics();
```

Comment:
Yes and no :slightly_smiling_face: Both are for monitoring. So it kind of make sense to put them together. At the same time, metrics configuration must be specific to metrics. May be we can split the configuration in to a httpserver config (enable httpServer always or by default) and metrics config. This would be a breaking change though.

Reply:
Can be done in a separate issue later on

Reply:
Created an issue for it #2652